src/Oro/BugBundle/Controller/IssueController.php

```diff
@@ -23,4 +26,120 @@ public function indexAction()
     {
         return ['entity_class' => 'Oro\BugBundle\Entity\Issue'];
     }
+
+
+    /**
+     * @Route("/issue/create", name="bug.issue_create")
+     * @Template("OroBugBundle:Issue:update.html.twig")
+     * @Acl(
+     *     id="bug.issue_create",
+     *     type="entity",
+     *     class="OroBugBundle:Issue",
+     *     permission="CREATE"
+     * )
+     * @return array|\Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function createAction()
+    {
+        return $this->update(new Issue());
+    }
+
+
+    /**
+     * @Route("/issue/create/{id}", name="bug.issue_children_create", requirements={"id":"\d+"}, defaults={"id":0})
+     * @Template("OroBugBundle:Issue:update.html.twig")
+     * @Acl(
```

Comment:
`@Acl` definition required only once
Please use @AclAncestor('bug.issue_create') after it was define